app/controllers/static_pages_controller.rb

```diff
@@ -3,7 +3,7 @@ class StaticPagesController < ApplicationController
                      only: [:api, :copyright, :home, :status]
 
   fine_print_skip :general_terms_of_use, :privacy_policy,
-                  only: [:api, :copyright, :home, :status]
```

Comment:
What happens if a user isn't logged in?  Does this change indirectly mean they'll be asked to login? (so they can eventually sign the agreements)